server/src/routes/bots/[id]/index.js

```diff
@@ -120,18 +121,30 @@ module.exports = {
       .optional()
       .isString().withMessage('Support server ID should be a string.')
       .isLength({ min: 17, max: 19 }).withMessage('Support server ID must be between 17 and 19 characters.'),
+    body('webhook')
+      .optional()
+      .isObject().withMessage('Webhook should be an object.')
+      .custom((webhook) => {
```

Comment:
You need to create a new validation file in /utils/validations/bots, move this validation function there and call it from there.